transcendence-app/src/shared/enums/action.enum.ts

```diff
@@ -0,0 +1,9 @@
+export enum Action {
```

Comment:
maybe we should split this in different types where all enums are meaningful in a given context?